OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -130,23 +130,23 @@ public void GetByProviderId_WhenIdIsInvalid_ShouldThrowArgumentOutOfRangeExcepti
         public async Task GetByProviderId_WhenThereAreWorkshops_ShouldReturnOkResultObject(long id)
         {
             // Arrange
-            workshopServiceMoq.Setup(x => x.GetByProviderId(id)).ReturnsAsync(workshops.Where(x => x.ProviderId == id));
+            workshopServiceMoq.Setup(x => x.GetByProviderId(id)).ReturnsAsync(workshopCards.Where(x => x.ProviderId == id));
 
             // Act
             var result = await controller.GetByProviderId(id).ConfigureAwait(false) as OkObjectResult;
 
             // Assert
             Assert.That(result, Is.Not.Null);
```

Comment:
I don't insist, just recommendation - not to mix 2 styles of `Assert`s: `.That` and `.Are...`. I know it's not scope of this PR, just discussion point

Reply:
Thank you for the advice